lib/qio/QIO_host_file_conversion.c

```diff
@@ -1081,7 +1087,7 @@ int QIO_part_to_single( const char filename[], int ildgstyle,
 	  /* Copy LIME type */
 	  lime_type_out = (char *)malloc(strlen(lime_type_in)+1);
 	  if( lime_type_out == NULL ) return QIO_BAD_ARG;
-	  sprintf(lime_type_out,"%s",lime_type_in);
+	  strcpy(lime_type_out,lime_type_in);
```

Comment:
random aside comment: are there other places in QIO that could have `sprintf` -> `strcpy` applied?  `sprintf` is notoriously slow compared to `strcpy`.